src/lib/FlightTasks/tasks/FlightTaskManualAltitudeSmooth.cpp

```diff
@@ -44,14 +44,16 @@ FlightTaskManualAltitudeSmooth::FlightTaskManualAltitudeSmooth(control::SuperBlo
 	_smoothing(_velocity(2), _sticks(2))
 {}
 
-void FlightTaskManualAltitudeSmooth::_updateSetpoints()
+
+void FlightTaskManualAltitudeSmooth::updateOutput(ControlSetpoint &setpoint)
```

Comment:
Hm, I mean it is really an architectural question. On the one hand, if I read this method it is clear what is going on. On the other hand, with this kind of implementation you need to copy paste in each output method what you have already implemented in a base class.